KIF Tests/OffscreenTests.m

```diff
@@ -0,0 +1,36 @@
+//
+//  OffscreenTests.m
+//  KIF Tests
+//
+//  Created by Steve Sun on 2023-03-28.
+//
+
+#import <KIF/KIF.h>
+#import "KIFTestStepValidation.h"
+
+@interface OffscreenTests : KIFTestCase
```

Comment:
We should either add an equivalent version of this against the KIFUIViewTestActor, or use that version, as that is what we'd like to be the first class API.